src/main/java/com/shopping_app/shoppingApp/service/AuthenticationService.java

```diff
@@ -0,0 +1,19 @@
+package com.shopping_app.shoppingApp.service;
+
+import com.shopping_app.shoppingApp.Exceptions.BaseException;
+import com.shopping_app.shoppingApp.utils.UserPrincipal;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthenticationService {
```

Comment:
can you explain how the name `AuthenticationService` is appropriate for this class ?

Reply:
here  i am verifying who a user is , so i named it as Authentication

Reply:
replaced with appropriate matching name